### PR TITLE
Add information on adding logs to Code Executor

### PR DESCRIPTION
**Pull request by sandesh89**
Add information about using standard python logger inside the Code Executor

### CODE REVIEW

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 1]
```diff
@@ -110,6 +110,20 @@ six-1.12.0
 unicodecsv-0.14.1 
 urllib3-1.25.3
 
+### Logger support in Code Executor
+You can use standard python logging library inside the code editor. See the below snippet for reference
+
+```python
+import  json
```
Comment (prik2693) [inline docs/bot-builder/integrations/integrating-custom-code.md]: add a line 
import logging

File: docs/bot-builder/integrations/integrating-custom-code.md [hunk 2]
```diff
@@ -110,6 +110,20 @@ six-1.12.0
...
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def main(event, context):
+    final_response = {'status': True}
+    logger.info(final_response)
```
Comment (prik2693) [inline docs/bot-builder/integrations/integrating-custom-code.md]: have we tested if logger.info is returned in the current lambda logs?

Comment (sandesh89) [reply docs/bot-builder/integrations/integrating-custom-code.md]: yes , you can check here.. feel free to play around on some other node and tell me if any strange behavior

https://staging.hellohaptik.com/mogambo/#/v2/bots/5ddbb433ef72780165731a77/stories/5ddbb45e7d3be40158c85126/substories/5ddbb4acbb65d501af4266f7/nodes/5ddbc4c04eed75001e6f1024/integrations/editor

File: docs/bot-builder/integrations/integrating-custom-code.md (hunk 1 above)
Comment (sandesh89) [reply docs/bot-builder/integrations/integrating-custom-code.md]: good catch. Updated.

File: docs/bot-builder/integrations/integrating-custom-code.md (hunk 2 above)
Comment (prik2693) [reply docs/bot-builder/integrations/integrating-custom-code.md]: great

**Pull request merged**